documentation/source/install/install_usecases.rst

```diff
@@ -354,3 +263,40 @@ It may take some time for the pods to start as the images must be pulled and ins
 The jupyter server dashboard can be accessed on the Kubernetes node port as in the following screen.
 
 .. image:: https://raw.githubusercontent.com/openebs/openebs/master/documentation/source/_static/Jupyter.png
+
+Crunchy Postgres
+=================
+ 
+Running Crunchy Postgres on OpenEBS
+------------------------------------
+
+The following steps bring up a postgresql stateful set with one master and one replica on OpenEBS storage. OpenEBS has used centos-based postgresql containers from crunchy-data to achieve the same.
+
+Download the files to your host, which has access to kubectl using the following commands.
+::
+  
+  cd $HOME
+  git clone https://github.com/openebs/openebs.git
+  cd openebs/k8s/demo/crunchy-postgres
+
+Run the StatefulSet using the following command. The files are available with default images and credentails (set.json)
+::
```

Comment:
The size of the openebs persistent storage volume is 400M by default. You can edit this in the storage class section of the specification file set.json 

cat set.json
..
"volumeClaimTemplates": [
      {
        "metadata": {
          "name": "pgdata"
        },
        "spec": {
          "accessModes": [
            "ReadWriteOnce"
          ],
	  "storageClassName": "openebs-basic",
          "resources": {
            "requests": {
              "storage": "400M"
            }
          }
        }
      }
..

Reply:
Done!